### PR TITLE
Added test_image option to convert command.

### PR DESCRIPTION
**Pull request by odoku**
## What this patch does to fix the issue.

#960

### CODE REVIEW

File: blueoil/cmd/convert.py [hunk 1]
```diff
@@ -137,6 +137,7 @@ def run(experiment_id,
         output_template_dir=None,
         image_size=(None, None),
         project_name=None,
+        test_image=DEFAULT_INFERENCE_TEST_DATA_IMAGE,
```
Comment (iizukak) [inline blueoil/cmd/convert.py:140]: Please change docstring.

**Comment by iizukak** [comment]
wow
@odoku Sorry, `blueoil/cmd/main.py` conflicted...

**Comment by odoku** [comment]
@iizukak I solved the conflict!!!

**Pull request merged**